internal/analysis/rundynamic.go

```diff
@@ -0,0 +1,111 @@
+package analysis
+
+import (
+	"github.com/ossf/package-analysis/internal/dynamicanalysis"
+	"github.com/ossf/package-analysis/internal/log"
+	"github.com/ossf/package-analysis/internal/pkgecosystem"
+	"github.com/ossf/package-analysis/internal/sandbox"
+)
+
+type DynamicAnalysisResults map[pkgecosystem.RunPhase]*dynamicanalysis.Result
+
+// RunDynamicAnalysis
+// Runs dynamic analysis on the given package in the sandbox provided, and returns the results and error (if any)
+// at all registered running phases for the package ecosystem. Status is logged to standard output if requested.
+// The return value maps each successfully run phase to the corresponding dynamic analysis result.
+func RunDynamicAnalysis(sb sandbox.Sandbox, pkg *pkgecosystem.Pkg, logStatus bool) (results DynamicAnalysisResults, err error) {
```

Comment:
I think logStatus can be removed. It is currently only set to "true"

Reply:
Fair, I wanted to at least give some control over side effects to the caller but don't feel too strongly

Reply:
Sure